configure.sh

```diff
@@ -26,6 +26,7 @@ General options;
   --win64                  cross-compile for Windows 64 bit
   --win64-native           natively compile for Windows 64 bit
   --ninja                  use Ninja build system
+  --ccache                 use ccache
```

Comment:
Is an option in the configure script really required, or can this also be done via setting CC and CXX?

Reply:
The use of `CMAKE_<LANG>_COMPILER_LAUNCHER=ccache` is critical for making it work. My first attempt was to change the PATH variable to point to symbolic links of ccache instead of the actual C/C++ compilers, as we do on Linux and macOS. However, this didn't work as expected because CMake couldn't identify the version of the compilers during configuration, leading to further issues during the build process.

Reply:
According to the [ccache documentation](https://github.com/ccache/ccache/wiki/CMake), these are the only two ways to use ccache with CMake.

Reply:
Changed.